### DISCUSSION TITLE
Microcontroller hangs after writing the downloaded bin file

### DISCUSSION THREAD

**Discussion by talhayousuff** [JAndrassy/ArduinoOTA#90] [Q&A]
Hey jandrassy, I have been able to implement ArduinoOTA with SIM800C on ATMEGA2560.

However, if I use it with Telit GL865, the microcontroller hangs after writing the downloaded bin file. 

I am using HTTPQRY and HTTPRCV commands to download the bin file from a remote server.

**Comment by JAndrassy** [comment]
I never worked with any GSM module with Arduino

**Reply by talhayousuff** [reply]
Thank you for responding.
